### ISSUE TITLE
Import doesn't work in Nuxt for some reason

### ISSUE THREAD

**Issue by watzon** [webistomin/vue-socials#571]
Trying to get vue-socials working with a Nuxt app. When I try to `import VueSocials from 'vue-socials'` in a plugin I get the following error:

```
Failed to resolve entry for package "vue-socials". The package may have incorrect main/module/exports specified in its package.json.
```

**Comment by webistomin** [comment]
Fixed in `v2.0.1`

**Comment by kaboume** [comment]
@watzon : wich Nuxt release do you use ? 
If Nuxt3, could you please share your plugin content ? 
Thanks.
